### PR TITLE
(#4471) Remove MLP Title Paragraph and add it to MLP Template

### LINKED ISSUE #4471: CR: Remove MLP Title Paragraph and Add it to the MLP Template
### Change Request
This request is to remove the MLP Title paragraph from Drupal and to add the title back to the template of Mini Landing Pages. 
Notable Proposed Changes (agreed upon by internal team 16OCT2024):
- The MLP Title paragraph will not be a paragraph to be added by content editors
- The MLP Title will appear above all other body content on MLP pages using the NCIDS style (including slim hero)
- Requirements change: 
  - The content editor shall **not** have the ability to control where the title is placed on MLPs

**Questions**
- How would we have to adjust the current measurement pages?
  - Only pages impacted are [Managing Cancer Care](https://www.cancer.gov/about-cancer/managing-care) and [Research Areas](https://www.cancer.gov/research/areas), do those need to be manually changed?
- Do we need to mark the change as a landmark in analytics testing to evaluate the change?

### DoR Tasks

- [x] Product (Andy) to gain approval from client on making this change
- [x] Design (@blilianyu @monika-jaeger) to make changes to comps to reflect the title going to the top (after gaining client approval)
- [x] Ticket, comp, and AC reviewed and approved by developer

### What's the expected change?
- The MLP pages with the NCIDS style will have the title appear at the top of the page above all body content.
- On the CMS, the MLP Title paragraph will be removed as a paragraph and it will be added to the template to render at the top of the page

### What's the current functionality?
- The MLP Title paragraph needs to be manually added to MLP pages using the NCIDS style, and it can be placed anywhere on the page (before or after the slim hero).

### What's the updated acceptance criteria?
**CMS Behavior**
```gherkin
Scenario: Adding or editing a new Mini Landing Page content type with the NCIDS Style on the CMS
  Given the user is logged in as an editor on the CMS
    And the user is adding or editing a "Mini Landing Page" content item
  When the user is on the Mini Landing Page content type
  Then the user sees the "Page Title" required field as a part of the Mini Landing Page template
  When the user adds text to the "Page Title"
  Then the Title will be added to the top of the page
    And there will be no paragraph to add a title to the Mini Landing Page
```

**Front-End Behavior**
```gherkin
Scenario: A site visitor goes to a Mini Landing Page using the NCIDS Style
  Given the user is on a Mini Landing Page
    And the Mini Landing Page is using the NCIDS Style
  When the user views the top of the page
  Then the page title will be displayed above all other body content
```

**Analytics**
- Any Mini Landing Page components that reference the page title shall maintain their functionality for analytics without change.

### Related Tickets
* Original Story Ticket #4357

### PR DESCRIPTION
**Pull request by dev-rana-publicis**
Closes #4471

### CODE REVIEW

File: docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/cgov_home_landing.install [hunk 1]
```diff
@@ -138,3 +138,29 @@ function cgov_home_landing_update_8002() {
...
+    ->accessCheck(FALSE)
+    ->execute();
+
+  if (!empty($ids)) {
+    $paragraphs = $storage->loadMultiple($ids);
+
+    // Delete existing entities.
+    $storage->delete($paragraphs);
+  }
+}
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/modules/custom/cgov_home_landing/cgov_home_landing.install:166]: Ok, so you still are not removing the configuration for the NCIDS Page Title Block.

File: docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/mini_landing/node--cgov-mini-landing--ncids-default--full.html.twig [hunk 2]
```diff
@@ -1,4 +1,7 @@
 <!-- Begin MAIN CONTENT AREA -->
 <main class="main-content type {{ page_type }}" id="main-content">
+  <div class="usa-section usa-section--cgdp-no-bottom" data-eddl-landing-row>
+    <h1 class="nci-heading-h1">{{ drupal_token('current-page:title')}}</h1>
```
Comment (bryanpizzillo) [inline docroot/profiles/custom/cgov_site/themes/custom/ncids_trans/templates/content/mini_landing/node--cgov-mini-landing--ncids-default--full.html.twig:4]: You should be able to just reference the node's title here, `<h1>{{ node.label }}</h1>`, and skip drupal_token. drupal_token should only be used when you can't get the data through a more direct (and more performant) way.

**Comment by bryanpizzillo** [comment]
I looked on www-dev-acsf and there are no errors in the error log, and the paragraph does not exist in structure > Paragraphs.

**Review by bryanpizzillo — APPROVED** [review]

**Review by andyvanavery31 — APPROVED** [review]
Passes product review.

**Pull request merged**